acceptance/detector_test.go

```diff
@@ -193,4 +193,125 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
+					h.WithArgs("-log-level=debug"),
+				)
+
+				// check group.toml
+				tempGroupToml := filepath.Join(copyDir, "layers", "group.toml")
+				var buildpackGroup lifecycle.BuildpackGroup
+				_, err := toml.DecodeFile(tempGroupToml, &buildpackGroup)
+				h.AssertNil(t, err)
+				h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
+				h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
+			})
+
+			when("and /layers/order.toml is present", func() {
+				it("prefers /layers/order.toml", func() {
+					h.DockerRunAndCopy(t,
+						containerName,
+						copyDir,
+						detectImage,
+						"/layers",
+						h.WithFlags("--user", userID,
+							"--volume", expectedOrderTOMLPath+":"+"/layers/order.toml",
+							"--volume", otherOrderTOMLPath+":"+"/cnb/order.toml",
+							"--env", "CNB_ORDER_PATH="),
+						h.WithArgs("-log-level=debug"),
+					)
+
+					// check group.toml
+					tempGroupToml := filepath.Join(copyDir, "layers", "group.toml")
+					var buildpackGroup lifecycle.BuildpackGroup
+					_, err := toml.DecodeFile(tempGroupToml, &buildpackGroup)
+					h.AssertNil(t, err)
+					h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
+					h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
+				})
+
+				when("platform api < 0.6", func() {
+					it("processes /cnb/order.toml", func() {
+						h.DockerRunAndCopy(t,
+							containerName,
+							copyDir,
+							detectImage,
+							"/layers",
+							h.WithFlags("--user", userID,
+								"--volume", expectedOrderTOMLPath+":"+"/cnb/order.toml",
+								"--volume", otherOrderTOMLPath+":"+"/layers/order.toml",
+								"--env", "CNB_PLATFORM_API=0.5",
+								"--env", "CNB_ORDER_PATH="),
+							h.WithArgs("-log-level=debug"),
+						)
+
+						// check group.toml
+						tempGroupToml := filepath.Join(copyDir, "layers", "group.toml")
+						var buildpackGroup lifecycle.BuildpackGroup
+						_, err := toml.DecodeFile(tempGroupToml, &buildpackGroup)
+						h.AssertNil(t, err)
+						h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
+						h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
+					})
+				})
+			})
+		})
+
+		when("/layers/order.toml is present", func() {
+			it("processes /layers/order.toml", func() {
+				h.DockerRunAndCopy(t,
+					containerName,
+					copyDir,
+					detectImage,
+					"/layers",
+					h.WithFlags("--user", userID,
+						"--volume", expectedOrderTOMLPath+":"+"/layers/order.toml",
+						"--env", "CNB_ORDER_PATH="),
+					h.WithArgs("-log-level=debug"),
+				)
+
+				// check group.toml
+				tempGroupToml := filepath.Join(copyDir, "layers", "group.toml")
+				var buildpackGroup lifecycle.BuildpackGroup
+				_, err := toml.DecodeFile(tempGroupToml, &buildpackGroup)
+				h.AssertNil(t, err)
+				h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
+				h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
+			})
+		})
```

Comment:
Is this context duplicating lines 245-265?

It might be more readable to pull the `when("platform api < 0.6"` context to be just under `when("/cnb/order.toml is present"`.

Reply:
Ohh I see - this one doesn't have `"--volume", otherOrderTOMLPath+":"+"/cnb/order.toml"`. TBH I'm not sure we need this context, but maybe I'm wrong...

Reply:
I went back and forth on this. I think I like it better where it is at, but if you feel strongly I can try and move things around.

Reply:
I'm okay leaving it where it is!